.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,22 @@
+- id: poetry-check
+  name: poetry-check
+  description: run poetry check to validate config
+  entry: poetry check
+  language: python
+  language_version: python3
+  pass_filenames: false
+- id: poetry-lock
```

Comment:
could you add a newline after each hook?

Reply:
Sure and done.